.stylelintrc.json

```diff
@@ -1,49 +1,50 @@
 {
-    "extends": "stylelint-config-standard",
-    "plugins": ["stylelint-scss", "stylelint-selector-bem-pattern"],
-    "ignoreFiles": ["**/_normalize.scss", "**/_cads-iconfont.scss"],
-    "rules": {
-        "at-rule-disallowed-list": ["debug"],
-        "at-rule-empty-line-before": [
-            "always",
-            {
-                "except": ["first-nested", "blockless-after-blockless"],
-                "ignore": ["after-comment"],
-                "ignoreAtRules": ["if", "else"]
-            }
-        ],
-        "at-rule-no-unknown": null,
-        "at-rule-no-vendor-prefix": true,
-        "block-closing-brace-newline-after": null,
-        "color-named": "never",
-        "declaration-colon-newline-after": null,
-        "indentation": 2,
-        "property-disallowed-list": ["transition"],
-        "rule-empty-line-before": [
-            "always-multi-line",
-            {
-                "except": ["first-nested"],
-                "ignore": ["after-comment"]
-            }
-        ],
-        "scss/at-if-closing-brace-newline-after": "always-last-in-chain",
-        "scss/at-function-pattern": "^[a-z]+([a-z0-9-]+[a-z0-9]+)?$",
-        "scss/at-import-no-partial-leading-underscore": true,
-        "scss/at-import-partial-extension-blacklist": ["scss"],
-        "scss/at-mixin-pattern": "^[a-z]+([a-z0-9-]+[a-z0-9]+)?$",
-        "scss/dollar-variable-colon-space-after": "always",
-        "scss/dollar-variable-colon-space-before": "never",
-        "scss/selector-no-redundant-nesting-selector": true,
-        "selector-max-compound-selectors": 6,
-        "selector-no-vendor-prefix": true,
-        "string-quotes": "single",
-        "value-no-vendor-prefix": true,
-        "plugin/selector-bem-pattern": {
-            "preset": "bem",
-            "componentName": "ds-(([a-z0-9]+(?!-$)-?)+)",
-            "componentSelectors": {
-                "initial": "\\.{componentName}(((__|--)(([a-z0-9\\[\\]'=]+(?!-$)-?)+))+)?$"
-            }
-        }
+  "extends": "stylelint-config-standard",
```

Comment:
Prettier auto-formatted this (from 4 spaces => 2 spaces). The notable change is to disable the pedantic `declaration-empty-line-before` rule. Particularly where there are larger numbers of styles, as in _buttons.scss, some deliberate spacing around properties is useful for grouping rules.

Reply:
Not too sure what ruleset you're after but `always first-nested` might be better than turning it off completely?  [https://stylelint.io/user-guide/rules/declaration-empty-line-before](https://stylelint.io/user-guide/rules/declaration-empty-line-before)